src/hermes/commands/harvest.py

```diff
@@ -16,6 +17,13 @@
 _CFF_VERSION = '1.2.0'
 
 
+def convert_cff_to_codemeta(cff_file):
```

Comment:
Maybe add type annotations...

Reply:
Done in f6190b1882d9cba79b692ab59888471fd16c50c8. Also fixed some errors that mypy reported.